### PR TITLE
# Data Access Framework (Upgraded .Net Framework version to 4.8)

### PR DESCRIPTION
**Pull request by vkgcodefactory**
Hi Vinod,
I have updated .Net framework version to 4.8, please review and merge it

Thanks
vkgcodefactory

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,266 @@
+# See http://help.github.com/ignore-files/ for more about ignoring files.
+
+# compiled output
+/dist
+/tmp
+/out-tsc
+
+# dependencies
+/node_modules
```
Comment (vkg-mca) [inline .gitignore]: remove /node_modules as it is not required here

File: DataAccessFramework/Readme.MD [hunk 2]
```diff
@@ -0,0 +1 @@
+# Data Access Framework (Upgraded .Net Framework version to 4.8 by vkgcodefactory)
```
Comment (vkg-mca) [inline DataAccessFramework/Readme.MD]: can you please add more details in the readme file to make more sense?

**Review by vkg-mca — CHANGES_REQUESTED** [review]

File: DataAccessFramework/Readme.MD (hunk 2 above)
Comment (vkg-mca) [reply DataAccessFramework/Readme.MD]: looks good

File: .gitignore (hunk 1 above)
Comment (vkg-mca) [reply .gitignore]: looks good, thanks

File: DataAccessFramework/Readme.MD (hunk 2 above)
Comment (vkg-mca) [reply DataAccessFramework/Readme.MD]: Can you elaborate more on the usage of the framework to the consumers

**Review by vkg-mca — CHANGES_REQUESTED** [review]
can you address the last comment, please?

**Comment by vkgcodefactory** [comment]
> can you address the last comment, please?

Sure, in the next version I will do, for now it will be great if you accept the changes

**Review by vkg-mca — APPROVED** [review]

**Comment by vkg-mca** [comment]
Looks good changes merged to master branch, thanks for your contribution

**Pull request merged**